app/src/main/java/com/android/shelfLife/di/FirestoreModule.kt

```diff
@@ -0,0 +1,19 @@
+// FirestoreModule.kt
+package com.android.shelfLife.di
+
+import com.google.firebase.auth.FirebaseAuth
+import com.google.firebase.firestore.FirebaseFirestore
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+object FirestoreModule {
+
+  @Singleton @Provides fun provideFirestore(): FirebaseFirestore = FirebaseFirestore.getInstance()
```

Comment:
Normally it is done by default but for security measures it would be good to specify the firebaseFirestore settings so that they work seamlessly offline. the code should look like this:
@Singleton
@Provides
fun provideFirestore(): FirebaseFirestore {
    val firestore = FirebaseFirestore.getInstance()
    // Ensure no Firestore operations have been performed yet
    val settings = com.google.firebase.firestore.FirebaseFirestoreSettings.Builder()
        .setPersistenceEnabled(true)
        .build()
    firestore.firestoreSettings = settings
    return firestore
}